kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskServiceIntegrationTest.java

```diff
@@ -247,6 +248,53 @@ public void testReleaseAndClaim() throws Exception {
             processClient.abortProcessInstance(CONTAINER_ID, processInstanceId);
         }
     }
+    
+    
+    @Test
+    public void testReleaseAndBulkClaim() throws Exception {
+        Long processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_USERTASK);
+        assertNotNull(processInstanceId);
+        assertTrue(processInstanceId.longValue() > 0);
+        try {
+            List<TaskSummary> taskList = taskClient.findTasksAssignedAsPotentialOwner(USER_YODA, 0, 10);
+            assertNotNull(taskList);
+
+            assertEquals(1, taskList.size());
+            TaskSummary taskSummary = taskList.get(0);
+            checkTaskNameAndStatus(taskSummary, "First task", Status.Reserved);
+
+            // release task
+            taskClient.releaseTask(CONTAINER_ID, taskSummary.getId(), USER_YODA);
+
+            taskList = taskClient.findTasksAssignedAsPotentialOwner(USER_YODA, 0, 10);
+            assertNotNull(taskList);
+
+            assertEquals(1, taskList.size());
+            taskSummary = taskList.get(0);
+            checkTaskNameAndStatus(taskSummary, "First task", Status.Ready);
+
+            taskClient.claimTask(CONTAINER_ID, Collections.singletonList(taskSummary.getId()), USER_YODA);
```

Comment:
Maybe I would try to claim more than 1 task to make sure we're reserving all the tasks passed as id, or combining with a non-claimable task to test it still works when one task cannot be reserved.

Reply:
Im not really sure this "implementation detail" should be checked by an existing test. 
As I said in other forum, I have serious doubts current silent approach (we are not communicating client the errors we are catching for individual operations) is the way to go. I implemented that way to no do differently to other bulk operations, but I prefer not enforce that in a test